nrpy/infrastructures/CarpetX/general_relativity/rhs_eval.py

```diff
@@ -0,0 +1,270 @@
+"""
+Generates function to compute the right-hand sides of the BSSN evolution equations.
+
+Authors: Zachariah B. Etienne
+        zachetie **at** gmail **dot* com
+         Samuel Cupp
+"""
+
+import re
+from collections import OrderedDict as ODict
+from typing import Union, cast, List
+from inspect import currentframe as cfr
+from types import FrameType as FT
+import sympy
+
+import nrpy.c_codegen as ccg
+import nrpy.c_function as cfc
+import nrpy.grid as gri
+import nrpy.params as par
+import nrpy.indexedexp as ixp
+import nrpy.helpers.parallel_codegen as pcg
+import nrpy.finite_difference as fin
+
+import nrpy.infrastructures.CarpetX.simple_loop as lp
+from nrpy.equations.general_relativity.BSSN_quantities import BSSN_quantities
+from nrpy.equations.general_relativity.BSSN_RHSs import BSSN_RHSs
+from nrpy.equations.general_relativity.BSSN_gauge_RHSs import BSSN_gauge_RHSs
+import nrpy.reference_metric as refmetric  # NRPy+: Reference metric support
+
+standard_ET_includes = [
```

Comment:
Centralize.

Reply:
Centralized in `CarpetX_include_header.py`